portal/models/auth.py

```diff
@@ -27,6 +29,9 @@ class AuthProvider(db.Model):
                         nullable=False)
     user = db.relationship('User')
 
+    __table_args__ = (UniqueConstraint('provider', 'provider_id',
+                                       name='auth_providers_by_provider'),)
```

Comment:
I prefer the single indentation too but was having trouble passing pep8squeaks and the auto indent tool I used did this. Will update.

Reply:
Actually, I'm not sure why I double indented this (and a few other places) because I was clearly passing pep8sqeaks with a single indent in other places. Nice catch.